harness/config/commands.yml

```diff
@@ -75,3 +75,18 @@ command('assets upload'):
   exec: |
     #!bash(workspace:/)|@
     passthru ws.aws s3 sync @('assets.local') @('assets.remote')
+
+command('frontend build'):
+  exec: |
+    #!bash(harness:/)|@
+    docker-compose -p @('namespace') run --rm -u node node sh -c 'cd @('frontend.path') && (test -f package.json || exit 0) && npm install && node_modules/.bin/gulp build'
```

Comment:
I could be wrong but i think the parentheses on the test run the command in a sub-shell meaning whether the file is there or not it will always do the npm install.

Reply:
You're right. We can just do `&& test -f package.json &&` as it's all strung together with && :)